gcloud-java-logging/src/main/java/com/google/cloud/logging/HttpRequest.java

```diff
@@ -458,7 +458,7 @@ public Builder toBuilder() {
     // todo(mziccard) uncomment when #1042 gets fixed
     // builder.setCacheLookup(cacheLookup);
     builder.setCacheHit(cacheHit);
-    builder.setValidatedWithOriginServer(cacheValidatedWithOriginServer);
+    builder.setCacheValidatedWithOriginServer(cacheValidatedWithOriginServer);
```

Comment:
TODOs in this class can be removed now (also in other classes) as I believe this fixes #1042. I can take care of doing that in a separate PR though.